bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/JsonParser.java

```diff
@@ -1,38 +1,61 @@
 /*
- *  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Do we need to change the licence header here?

Reply:
This is a newly added file. I updated the class name as the old `JsonParser` class is no longer needed.

Reply:
I updated the header to the previous format as we use the old class name.